packages/oas-normalize/src/index.ts

```diff
@@ -84,8 +84,10 @@ export default class OASNormalize {
   /**
    * @private
    */
-  static convertPostmanToOpenAPI(schema: any) {
-    return postmanToOpenAPI(JSON.stringify(schema), null, { outputFormat: 'json', replaceVars: true }).then(JSON.parse);
+  static async convertPostmanToOpenAPI(schema: any) {
```

Comment:
added `async` here to address this warning:

![CleanShot 2024-04-09 at 14 06 59@2x](https://github.com/readmeio/oas/assets/8854718/aa132ee7-7640-4187-a4c3-3c4b9f5cc0dc)

wish there was a way for `tsc` to actually throw on these things 🧐